build/config.go

```diff
@@ -59,3 +61,10 @@ func (cfg *LoggerConfig) HandlerOptions() []btclog.HandlerOption {
 
 	return opts
 }
+
+// FileLoggerConfig extends LoggerConfig with specific log file options.
+//
+//nolint:lll
```

Comment:
can be removed now?

Reply:
Wanted to comment the same when going through it commit by commit. But longer lines are added to it later, so I think it's still needed.

Reply:
oh yeah lol 🙃 will re-add (in correct commit)

Reply:
oh ok didn't notice that😂